src/Models/ReflectionContainer.php

```diff
@@ -28,6 +28,12 @@ public function dataObjectPropertyColumns(
         array $props
     ): array
     {
+        // Empty props should result in an empty array.
+        if (!count($props))
+        {
+            return [];
+        }
+
```

Comment:
This check may not be necessary but I can see somewhere in the future someone trying to brute force this by passing in an empty array.